content/W2015/W15-event-boardgames.md

```diff
@@ -13,6 +13,8 @@ some new faces.
 
 Hope to see you there!
 
+Our Big CSters program helps connect ciswomen, transwomen, and non-binary students with other students in CS and related fields, through social events geared towards mentorship.
```

Comment:
I'll keep the way it is currently though and leave the "and"s for further discussion :)